tests/ui/redundant_pattern_matching.stderr

```diff
@@ -1,31 +1,31 @@
 error: redundant pattern matching, consider using `is_ok()`
-  --> $DIR/redundant_pattern_matching.rs:5:12
+  --> $DIR/redundant_pattern_matching.rs:6:12
    |
 LL |     if let Ok(_) = Ok::<i32, i32>(42) {}
-   |     -------^^^^^------------------------ help: try this: `if Ok::<i32, i32>(42).is_ok()`
+   |     -------^^^^^------------------------ help: try this: `Ok::<i32, i32>(42).is_ok()`
```

Comment:
Not for this PR, but applying this suggestion results in a syntax error, since the `;` is missing. Not sure how to solve this, since on a `let _ = if ...;` the `;` is already there, on a returning `if` the `;` shouldn't be added, and on a stray `if` it has to be added.